### PR TITLE
[3.9][plg_user_terms] Add namespaces

### PR DESCRIPTION
**Pull request by Septdir**
At first, I made PR by accident. I apologize for this. From now on I will be more attentive.
I decided that it would just be wrong to close and decided to add missing namespaces to User - Terms plugin.

### Testing Instructions
Plugin should work correctly after applying this patch.

### CODE REVIEW

**Comment by Septdir** [comment]
Ups

**Review by laoneo — APPROVED** [review]

File: plugins/user/terms/field/terms.php [hunk 1]
```diff
@@ -9,10 +9,14 @@
 
 defined('JPATH_PLATFORM') or die;
 
+use Joomla\CMS\Form\FormHelper;
```
Comment (Quy) [inline plugins/user/terms/field/terms.php]: Please sort use statements in alpha order.

Comment (Septdir) [reply plugins/user/terms/field/terms.php]: Ok

**Comment by Septdir** [comment]
@Quy check Alpha order check please.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6c635009b1c79e68f20e360cf577db2ad83f8084<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22847">issues.joomla.org/tracker/joomla-cms/22847</a>.</sub>

**Comment by pavluk** [comment]
I have tested this item :white_check_mark: successfully on 6c635009b1c79e68f20e360cf577db2ad83f8084<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22847">issues.joomla.org/tracker/joomla-cms/22847</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22847">issues.joomla.org/tracker/joomla-cms/22847</a>.</sub>

**Pull request merged**